src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -16,6 +21,11 @@ const doSearch = async (term: string) => {
     })
   )
 
+  sendAnalyticsEvent<SearchEvent>({
+    name: 'search',
+    params: { search_term: term },
```

Comment:
Just out of curiosity, is this the only attribute required/supported for the `search` event?

Reply:
Apparently it is, Heitor. https://developers.google.com/analytics/devguides/collection/ga4/reference/events#search